File: java/src/main/java/deltix/luminary/LiteralNull.java
package deltix.luminary;

public class LiteralNull extends Literal {
    private LiteralNull() {
        super(LiteralKind.NULL);
    }

    public Object getValue() {
        return null;
    }

    @Override
    public String toString() {
        return "null";
    }

    public final static LiteralNull INSTANCE = new LiteralNull();
}

File: csharp/main/LiteralNull.cs
using System;

namespace Deltix.Luminary
{
	public class LiteralNull : Literal
	{
		private LiteralNull()
			: base(LiteralKind.Null)
		{}

		public override String ToString()
		{
			return "null";
		}

		public static readonly LiteralNull Instance = new LiteralNull();
	}
}
